compiler/concurrent_test.go

```diff
@@ -232,11 +181,33 @@ func TestAutoGenerateConcurrent_HandlesPackageWithNoOutput(t *testing.T) {
 	pkgDir := filepath.Join(mroot, "compiler")
 	pkgs := loadPackages(t, pkgDir)
 	module, err := genieql.FindModulePath(pkgDir)
-	if err != nil {
-		t.Fatalf("failed to find module path: %v", err)
-	}
+	require.NoError(t, err, "failed to find module path")
 	err = compiler.AutoGenerateConcurrent(testctx, "postgresql.test.config", bctx, module, defaultOutputFilename, pkgs)
-	if err != nil {
-		t.Fatalf("AutoGenerateConcurrent failed: %v", err)
+	require.NoError(t, err, "AutoGenerateConcurrent failed")
+}
+
+func TestAutoCompileGraph_WithBuildContextDirSet(t *testing.T) {
+	testctx, bctx, mroot := setupTest(t)
+	pkgDir := filepath.Join(mroot, "examples/postgresql/autocompilegraph")
+	bctx.Dir = pkgDir
+	pkgs := loadPackages(t, pkgDir+"/...")
+	module, err := genieql.FindModulePath(pkgDir)
+	require.NoError(t, err, "failed to find module path")
+	results, err := compiler.AutoCompileGraph(testctx, "postgresql.test.config", bctx, module, defaultOutputFilename, pkgs)
```

Comment:
default.config will work for your tests. just use that.
```suggestion
	results, err := compiler.AutoCompileGraph(testctx, "default.config", bctx, module, defaultOutputFilename, pkgs)
```